### DISCUSSION TITLE
openai_key 接触错误，是网络问题还是别的？

### DISCUSSION THREAD

**Discussion by harukasky** [JimmyLv/BibiGPT#87] [Q&A]
本地运行是可以的
如图
![image](https://user-images.githubusercontent.com/129709601/229964651-5943c205-cb91-43c6-9435-c78a603c8f6c.png)
但是openai_key无法接触到，一直说fetch failed，有什么好的解决办法吗？

**Comment by JimmyLv** [comment]
你添加了 .env 文件么？

**Comment by harukasky** [comment]
你好，添加了的

**Comment by harukasky** [comment]
所需要的环境变量都配置了，但是始终无法接触到OPENAI_API_KEY，显示
{ apiKey: 'sk-********************************' }
fetch failed
YouTube上的视频解析以及cmd窗口命令，如下
![image](https://user-images.githubusercontent.com/129709601/229993184-1ab3d2f4-b728-4f08-a910-2a12eddcc82c.png)
![image](https://user-images.githubusercontent.com/129709601/229993603-9d155262-9621-4df3-902c-ea1dce0055ef.png)

**Comment by JimmyLv** [comment]
看报错的话，获取YouTube字幕是一个第三方平台，需要配置额外的key的
